test/test-errortracker.js

```diff
@@ -50,24 +51,12 @@ describe('Test how server responds to requests', function() {
     sinon.stub(Math, 'random').callsFake(function() {
       return randomVal;
     });
+    sinon.stub(log, 'write').yields(false);
   });
 
   after(function() {
     Math.random.restore();
-  });
-
-  it('Should ignore 99% of user errors', function() {
-    randomVal = 1;
-    query.a = 1;
-    query.ca = 0;
-    query.rt = '';
-    query['3p'] = 0;
-    query.s = '  at new vi (https://cdn.ampproject.org/rtv/031496877433269/v0.js:297:149)';
-    return chai.request(app).get('/r').query(query).then(function(res) {
-      expect(res).to.have.header('Content-Type', 'text/plain; charset=utf-8');
-      expect(res).to.have.status(statusCodes.OK);
-      expect(res.text).to.equal('THROTTLED\n');
-    });
+    log.write.restore();
```

Comment:
If you use a sandbox, you won't have to manually restore (which avoids bugs due to forgetting).

```js
const sandbox = sinon.sandbox.create();
sandbox.stub(...);
```